types/block.go

```diff
@@ -283,12 +283,12 @@ func BlockFromProto(bp *tmproto.Block) (*Block, error) {
 // MaxDataBytes returns the maximum size of block's data.
 //
 // XXX: Panics on negative result.
-func MaxDataBytes(maxBytes int64, valsCount, evidenceCount int) int64 {
+func MaxDataBytes(maxBytes int64, keyType crypto.KeyType, valsCount, evidenceCount int) int64 {
```

Comment:
Why do we need a key type provided in these?

Reply:
Same reason as stated above.